op-chain-ops/genesis/layer_two.go

```diff
@@ -28,6 +28,8 @@ const (
 	L2AllocsGranite  L2AllocsMode = "granite"
 	L2AllocsHolocene L2AllocsMode = "holocene"
 	L2AllocsIsthmus  L2AllocsMode = "isthmus"
+	// This needs to be updated whenever the latest hardfork is changed.
+	LatestHF L2AllocsMode = L2AllocsIsthmus
```

Comment:
I wonder if "latest hardfork" should be the latest code complete one, or the latest one in development?

Either way we get test failures with this change which need investigating.

Reply:
Maybe the former is more practical, I changed it to holocene in b604cb21c28257da09ff145608f635e87c099c1f.